### ISSUE TITLE
Missing data for chap_7/tensorboard/data

### ISSUE THREAD

**Issue by fluentcodes** [deeplearning-mit-tensorflow-keras-tfjs/2020_Zweite_Auflage#9]
Hi, 
I've just started with chapter 7 tensorboard but the example has missing file I can't find on 
https://github.com/deeplearning-mit-tensorflow-keras-tfjs/2020_Zweite_Auflage/tree/master/Quellcode/chap_7/tensorboard/data
Thank you for a quick response since its part of a course. 
Werner

    with gzip.open("train-labels-idx1-ubyte.gz") as path:
        train_labels = np.frombuffer(path.read(),np.uint8,offset=8)

    ## Bilder (Eingabe)
    with gzip.open("train-images-idx3-ubyte.gz") as path:
        train_images = np.frombuffer(path.read(),np.uint8,offset=16).reshape(len(train_labels),28,28)/255.0

    # Testdaten 

    ## Labels (Ausgabe)
    with gzip.open("t10k-labels-idx1-ubyte.gz") as path:
        test_labels = np.frombuffer(path.read(),np.uint8,offset=8)

    ## Bilder (Eingabe)
    with gzip.open("t10k-images-idx3-ubyte.gz") as path:
        test_images = np.frombuffer(path.read(),np.uint8,offset=16).reshape(len(test_labels),28,28)/255.0
